[12.0][ADD] web_widget_numeric_step (#1515)
OCA/web: web_widget_numeric_step/readme/ROADMAP.rst

> * The widget doesn't check the min and/or max value(s), use incorrect values for that attributes will provoke unexpected behaviors

Reviewer: ```suggestion
* The widget doesn't check the min and/or max value(s), using incorrect values for those attributes will provoke unexpected behaviours
```
